Code/Nancy.SimpleAuthentication/SimpleAuthenticationModule.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Nancy.SimpleAuthentication
 {
-    public class SimpleAuthenticationModule : NancyModule
+    public sealed class SimpleAuthenticationModule : NancyModule

```

Comment:
why `sealed`? (in know what `sealed` keyword is/does). what are you trying to communicate/demonstrate, here?

Reply:
It removes the warning on using the virtual `Get` / `Post` methods in the ctor.

Reply:
Ta. TIL.